### PR TITLE
DAOS-4956 doc: update control plane server readmes

### PR DESCRIPTION
**Pull request by tanabarr**
Doc-only: true

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

**Review by cdurf1 — APPROVED** [review]

**Review by cdurf1 — DISMISSED** [review]

File: doc/admin/deployment.md [hunk 1]
```diff
@@ -361,30 +361,31 @@ storage nodes via the dmg utility.
...
 `sudo daos_server storage prepare --scm-only`.
 
-The first time the command is run, the SCM AppDirect regions will be created as
-resource allocations on any available DCPM modules (one region per NUMA
+The first time the command is run, the SCM interleaved regions will be created
+as resource allocations on any available PMem modules (one region per NUMA
 node/socket). The regions are activated after BIOS reads the new resource
-allocations, and after initial completion the command prints a
-message to ask for a reboot (the command will not initiate reboot itself).
+allocations, and after initial completion the command prints a message to ask
```
Comment (jbrosenz) [inline doc/admin/deployment.md:403]: Perhaps:  ```Upon completion, the storage prepare command will prompt the admin to reboot the storage node(s) in order for the BIOS to activate these new regions.  The storage prepare command does not initiate the reboot itself.```

File: src/control/README.md [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
+
+The pbin package provides a framework for forwarding of requests to be executed
+by the privileged binary `daos_admin` on behalf of `daos_server`.
 
 The provider package contains interface shims to the external environment,
 initially just to the Linux operating system.
 
-The system package encapsulates the concept of the DAOS system, and
-it's associated membership.
+The system package encapsulates the concept of the DAOS system, and it's
```
Comment (jbrosenz) [inline src/control/README.md:35]: "it's" should be "its"

File: src/control/lib/control/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+then it is handled.
+If not, those hosts will respond with a structured error (system.ErrNotReplica
+or system.ErrNotLeader) that contains hints as to where the client should try
+next.
+Non-replica hosts respond with ErrNotReplica and replicas which are not the
+leader respond with ErrNotLeader.
+In the worst-case scenario a MS request could require 2 RPCs to service.
+For server-to-server MS requests, the process is more efficient because each
+server has the list of access points in it's config file, it can skip the first
+shotgun RPC and instead send to the APs, one of which should be able to service
```
Comment (jbrosenz) [inline src/control/lib/control/README.md:28]: I am not sure how well 'shotgun' will translate to non-native English speakers.  The idiom may be lost.  Perhaps there's a way to say this without the idiom.

File: src/control/lib/netdetect/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Netdetect library
```
Comment (jbrosenz) [inline src/control/lib/netdetect/README.md:1]: It wasn't on my radar that we wanted such a section for the netdetect library.   I am not entirely clear on the goal of the section (for example, who the target audience is and then what we want them to really know).

**Review by jbrosenz — APPROVED** [review]
LGTM.

**Review by kjacque — COMMENTED** [review]
A few minor comments, thanks for taking care of all these doc edits!

File: src/control/cmd/daos_server/README.md [hunk 5]
```diff
@@ -1,67 +1,60 @@
...
 
-The DAOS Server (control-plane) can be run either as a foreground
-process to issue hardware provisioning commands or as a background
-process (binary name followed by the "start" subcommand) when it will
-listen for instructions from the
-[management tool](/src/control/cmd/dmg/README.md) and manage bring-up
-of the DAOS system.  A DAOS Server instance will typically be run on
-every storage node to orchestrate the operation of the managed DAOS
-Engine processes.
+The DAOS Server (control-plane) can be run either as a foreground process to
```
Comment (mjmac) [inline src/control/cmd/daos_server/README.md:3]: (minor nit) The hyphenation of control/data plane in this file is inconsistent with the wording in src/control/README.md

File: doc/admin/deployment.md [hunk 6]
```diff
@@ -822,11 +893,15 @@ localhost
...
 
 To aid in provider configuration and debug, it may be helpful to run the
-fi_pingpong test (delivered as part of OFI/libfabric).  To run that test,
-determine the name of the provider to test usually by removing the "ofi+" prefix from the network scan provider data. Do use the "ofi+" prefix in the
-daos_server.yml. Do not use the "ofi+" prefix with fi_pingpong.
+`fi_pingpong` test (delivered as part of OFI/libfabric).
+To run that test, determine the name of the provider to test usually by
+removing the "ofi+" prefix from the network scan provider data.
+Do use the "ofi+" prefix in the `daos_server.yml`. Do not use the "ofi+" prefix
+with `fi_pingpong`.
```
[the remark this answers is not in the document]
Comment (tanabarr) [reply doc/admin/deployment.md:900]: I didn't write it but I will update

File: src/control/lib/control/README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# DAOS Control API
+
+The control package exposes an RPC-based API for control-plane client
```
Comment (mjmac) [inline src/control/lib/control/README.md:3]: Also here.

File: src/control/lib/control/README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+subset of the entire hostlist, if one of those hosts can service the request
+then it is handled.
+If not, those hosts will respond with a structured error (system.ErrNotReplica
+or system.ErrNotLeader) that contains hints as to where the client should try
+next.
+Non-replica hosts respond with ErrNotReplica and replicas which are not the
+leader respond with ErrNotLeader.
+In the worst-case scenario a MS request could require 2 RPCs to service.
+For server-to-server MS requests, the process is more efficient because each
+server has the list of access points in it's config file, it can skip the first
```
Comment (mjmac) [inline src/control/lib/control/README.md:27]: (nit) it's -> its

**Review by mjmac — COMMENTED** [review]
Looks good to me, nice to have updated docs. Just a couple of minor nits that I spotted.

**Review by mjmac — APPROVED** [review]
LGTM

**Review by kjacque — APPROVED** [review]

**Pull request merged**